ido/src/modules/project.rs

```diff
@@ -358,8 +358,25 @@ impl IDOContract {
         };
     }
 
+    pub(crate) fn internal_commit_ft_token(&mut self,account_id:AccountId, project_id: ProjectId, fund_contract_id: AccountId, deposit: U128, memo: String)-> U128{
+        let project = self.internal_get_project_or_panic(project_id);
+        
+        let buy_amount = self.internal_sale_commit(project_id,&account_id,deposit.0,memo);
+            
+        env::log(format!("Total buyed: {}",buy_amount ).as_bytes());
+
+        ext_ft_contract::ft_transfer(
```

Comment:
Again, we don't transfer the funding amount (ft token) to the project owner at this time. We will write a new function for the project owner to do that at the distribution time.

Please remove this and only use `internal_commit`.